arkouda_proxy_server/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "arkouda_proxy_server"
+version = "0.1.0"
+edition = "2021"
```

Comment:
idk if we wanna update this or keep it the same as arkouda?

Reply:
ah, good point, makes sense, I'll do dat

Reply:
only issue is that the format is 2023.6.16, so cannot do v2023.06.16.

Reply:
@pierce314159 let's keep the arkouda_proxxy_server version separate and I will look into how I can specify the supported arkouda versions